### PR TITLE
Nc news 7

### PR DESCRIPTION
**Pull request by nmb882003**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -47,6 +47,19 @@ describe(`GET /api/articles/:article_id`, () => {
             }));
         });
     });
+    test(`status:200, responds with an article object containing a 'comment-count' property`, () => {
+        return request(app)
+        .get(`/api/articles/1`)
+        .expect(200)
+        .then(({body}) => {
+            const {article} = body;
+            console.log(article);
```
Comment (daviidbartlett) [inline __tests__/app.test.js]: remove console.log

File: models/models.js [hunk 2]
```diff
@@ -6,9 +6,18 @@ exports.extractTopics = () => {
 };
 
 exports.extractArticleById = (article_id) => {
-    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
-    .then(({rows}) => {
-        if (rows.length) return rows[0];
+    const articlePromise = db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id]);
+    const commentsPromise = db.query(`SELECT * FROM comments WHERE article_id = $1;`, [article_id]);
+
```
Comment (daviidbartlett) [inline models/models.js:10]: this works, but shouldn't need to do this with two queries. Take a look at the SQL notes for how to do this with just one

File: __tests__/app.test.js (hunk 1 above)
Comment (nmb882003) [reply __tests__/app.test.js]: It is no more :'(

File: models/models.js (hunk 2 above)
Comment (nmb882003) [reply models/models.js:10]: Done! And still reeling in shock from the fact that it worked on the second attempt after I managed to find the relevant section in the notes

File: __tests__/app.test.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ describe(`GET /api/topics`, () => {
     })
 });
 
-describe(`GET /api/articles/:article_id`, () => {
+describe.only(`GET /api/articles/:article_id`, () => {
```
Comment (dappernerddesigns) [inline __tests__/app.test.js]: You must remove any .only, xtest, xit before submitting a pull request. How can we be sure these changes do not break earlier code?

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Please remove your .only, re run the tests, re commit and push. Then drop me a DM to double check your PR. We cannot check any PR with a .only in the test file.

File: __tests__/app.test.js (hunk 3 above)
Comment (nmb882003) [reply __tests__/app.test.js]: Gah! I literally spotted it and re-pushed just as you were writing this... On it!

**Review by dappernerddesigns — APPROVED** [review]
Nice refactor. Always remember to check for rogue console logs and .only's :D

**Pull request merged**